imx/examples/collection/.env.example

```diff
@@ -5,8 +5,8 @@
 # Etherum Network
 #=============================================
 ALCHEMY_API_KEY=
-# ropsten/mainnet etc.
-ETH_NETWORK=ropsten
+# sandbox/mainnet etc.
+ETH_NETWORK=sandbox
```

Comment:
The `ETH_NETWORK` env variable is defined in the examples but never used. I can't find reference to it anywhere.
Do you still need it in the `.env.example` files?

Reply:
Addressed this to be used by examples.